coffee-chats/src/main/webapp/src/components/NavBar.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import {AppBar, Grid, Icon, IconButton, Toolbar, Tooltip} from "@material-ui/core";
+
+export function NavBar() {
+  return (
+      <AppBar position="static" style={{ background: "transparent", boxShadow: "none"}}>
+        <Toolbar>
+          <Grid justify="space-between" container>
+            <Grid item>
+            </Grid>
```

Comment:
Is there a reasoning behind the empty grid item?

Reply:
Apparently that's how you align stuff to the right in our modern flexbox world